pkgs/applications/science/biology/svaba/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, zlib, bzip2, lzma, fetchFromGitHub } :
+
+stdenv.mkDerivation rec {
+  version = "1.1.0";
+  pname = "svaba";
+
+  src = fetchFromGitHub {
+    owner = "walaj";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "1vv5mc9z5d22kgdy7mm27ya5aahnqgkcrskdr2405058ikk9g8kp";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [ zlib bzip2 lzma ];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm555 src/svaba/svaba $out/bin/svaba
+    runHook postInstal
+  '';
```

Comment:
remove, replace with `autoreconfHook`

Reply:
The build just completed, and it looks like it didn't install at the right prefix, nevermind, keep the original installPhase

Reply:
disregard, autoconfHook, doesn't add anything in this case.